week-1/implement/sum.js

```diff
@@ -0,0 +1,20 @@
+
+function sum(input) {
+  let output = [];
+  for (let i = 0; i < input.length; i++) {
+    if (typeof input[i] !== "string") {
+      output.push(input[i]);
+    }
+  }
+  let total = 0;
+  for (let i = 0; i < output.length; i++){
+    total = total + output[i];
+  }
+  return total;
```

Comment:
This works, but it could be refactored and done in just a few lines - have a look at the reduce() fuction https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce